plaid/resources/plaid.toml

```diff
@@ -33,7 +33,7 @@ okta = 9_000_000
 "example_rule.wasm" = 5_000_000
 
 [loading.memory_page_count]
-default = 250
+default = 300
```

Comment:
Tested with a repository which had 361 collaborators, and I was hitting the memory limit.